website/public/locales/en.json

```diff
@@ -66,24 +66,24 @@
   },
   "expiration": {
     "legend": "The encrypted message will be deleted automatically after",
-    "optionOneHourLabel": "One Hour",
-    "optionOneDayLabel": "One Day",
-    "optionOneWeekLabel": "One Week"
+    "optionOneHourLabel": "One hour",
+    "optionOneDayLabel": "One day",
+    "optionOneWeekLabel": "One week"
   },
   "features": {
```

Comment:
It might be cleaner to keep title casing on the feature titles.

Reply:
Title casing on the features and sentence casing everywhere else? It looks weird when casings are mixed too me. Like when a codebase doesn't follow a consistent style.